frontend/src/assets/content/arp/contact-us.md

```diff
@@ -10,7 +10,7 @@ Fort Collins, CO 80526
 
 email: [clrdvis@fs.fed.us](mailto:clrdvis@fs.fed.us)
 
-Permits are available for purchase in person at the Canyon Lakes office Dec. 1 - Dec. 9, 2018. The office is closed on Wednesdays. Permits for the Red Feather cutting area are also available on-site the first two weekends in December.
+Permits are available for purchase in person at the Canyon Lakes office Nov. 25 - Dec. 13, 2019. The office is closed on Wednesdays. Permits for the Red Feather cutting area are also available on-site the first two weekends in December.
```

Comment:
looks like we were hardcoding the values here before your change, but just wanted to point it out bc i would've expected to see `{{startDate}}` and `{{endDate}}` here

Reply:
this cutting area will remain hard-coded. It has no code associated to it and permit purchase for it is manual.